iset.mm

```diff
@@ -72342,6 +72364,156 @@ Similar to the Axiom of Choice (first form) of [Enderton] p. 49.
   $}
 
 
+$(
+=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
+  Apartness relations
+=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
+$)
+
+  $c TAp $.
+
+  $( Tight apartness predicate symbol. $)
+  wtap $a wff R TAp A $.
+
+  ${
+    $d x y z A $.  $d x y z R $.
+    $( Tight apartness predicate.  A relation ` R ` is a tight apartness if it
+       is irreflexive, symmetric, cotransitive, and tight.  (Contributed by Jim
+       Kingdon, 5-Feb-2025.) $)
+    df-tap $a |- ( R TAp A <-> ( R C_ ( A X. A ) /\ ( A. x e. A -. x R x /\
+      A. x e. A A. y e. A ( x R y -> y R x ) ) /\
+      ( A. x e. A A. y e. A A. z e. A ( x R y -> ( x R z \/ y R z ) ) /\
+      A. x e. A A. y e. A ( -. x R y -> x = y ) ) ) ) $.
+  $}
+
+  ${
+    $d A x y z $.  $d R x y z $.  $d S x y z $.
+    $( Equality theorem for tight apartness predicate.  (Contributed by Jim
+       Kingdon, 8-Feb-2025.) $)
+    tapeq1 $p |- ( R = S -> ( R TAp A <-> S TAp A ) ) $=
+      ( vx vy vz wceq wss cv wbr wn wral wi wa wo w3a wtap breq notbid 2ralbidv
+      cxp sseq1 ralbidv imbi12d anbi12d orbi12d imbi1d 3anbi123d df-tap 3bitr4g
+      ) BCGZBAAUAZHZDIZUNBJZKZDALZUNEIZBJZURUNBJZMZEALDALZNZUSUNFIZBJZURVDBJZOZ
+      MZFALZEALDALZUSKZUNURGZMZEALDALZNZPCULHZUNUNCJZKZDALZUNURCJZURUNCJZMZEALD
+      ALZNZVTUNVDCJZURVDCJZOZMZFALZEALDALZVTKZVLMZEALDALZNZPABQACQUKUMVPVCWDVOW
+      NBCULUBUKUQVSVBWCUKUPVRDAUKUOVQUNUNBCRSUCUKVAWBDEAAUKUSVTUTWAUNURBCRZURUN
+      BCRUDTUEUKVJWJVNWMUKVIWIDEAAUKVHWHFAUKUSVTVGWGWOUKVEWEVFWFUNVDBCRURVDBCRU
+      FUDUCTUKVMWLDEAAUKVKWKVLUKUSVTWOSUGTUEUHDEFABUIDEFACUIUJ $.
+  $}
+
+  ${
+    $d A a b c u v $.  $d A a b c x y $.
+    $( Negated equality on a set with decidable equality is a tight apartness.
+       (Contributed by Jim Kingdon, 5-Feb-2025.) $)
```

Comment:
Wouldn't it make it clearer to have the notion of apartness relation first, and then of a tight one ?  Then, an intermediate result that inequality is an apartness relation may make is easier to follow ? (I haven't checked.)

Reply:
I thought about that and a bunch of other things which could be done. Seemed like a lot to take on for this pull request (which is very much focused on the thing which was recently formalized in TypeTopology).

Wouldn't rule it out for the future, though. Don't think I have any opinions firm enough to have all that much of a reaction at this point.